inference-engine/tests_deprecated/functional/vpu/common/myriad_hw_conv_tests.hpp

```diff
@@ -147,17 +145,17 @@ TEST_P(MyriadX_HW_Convolution_Tests_nightly, WithClamp) {
         SKIP() << "Non-MyriadX device";
     }
 
-    // TODO: Issue: 34466
-    if (CheckMA2085()) {
-        const ::testing::TestInfo* const test_info =
-              ::testing::UnitTest::GetInstance()->current_test_info();
-        const char* test_name = test_info->name();
-        const char* test_case = test_info->test_case_name();
-        if (!strcmp(test_name, "WithClamp/0") &&
-            !strcmp(test_case, "conv_3x3s1p1_vgg/MyriadX_HW_Convolution_Tests_nightly")) {
-            SKIP() << "Issue: 34466";
-        }
-    }
+//    // TODO: Issue: 34466
+//    if (CheckMA2085()) {
+//        const ::testing::TestInfo* const test_info =
+//              ::testing::UnitTest::GetInstance()->current_test_info();
+//        const char* test_name = test_info->name();
+//        const char* test_case = test_info->test_case_name();
+//        if (!strcmp(test_name, "WithClamp/0") &&
+//            !strcmp(test_case, "conv_3x3s1p1_vgg/MyriadX_HW_Convolution_Tests_nightly")) {
+//            SKIP() << "Issue: 34466";
+//        }
+//    }
```

Comment:
@nikita-kud  
Why do you comment these lines? (why not remove?)

Anyway, if you leave these lines (a;though commented), then could you please leave the line 9 as well (maybe also commented)? (otherwise this piece of code would not work if someone occasionally un-comment it)

Reply:
@elatkin I'm sorry. I just forgot to remove these lines after changing the status from "draft" to "ready"